docs/Data_Sources_Catalog/druid.md

```diff
@@ -3,13 +3,13 @@ sidebar_position: 5
 id: 'druid'
 ---
 
-# Druid
+# Druid (Experimental)
 
 :::info
 
 - Current usecase for <strong>Druid</strong> is with <strong>Roll-up</strong> enabled only.
 - Supports only `COUNT` and `SUM` aggregation. We have future plans for `MEAN` and `UNIQUE`.
-- We assume the name for the <strong>Count Column</strong> (that Druid creates when Roll-up is enabled) is `count`, this is  the default name that Druid assigns. We plan to make this a configurable parameter for datasources where preaggregation or materialised views are available.
+- Druid based KPIs require name for the <strong>Count Column</strong> for the rolled-up data (this column can also be proxy for count for the rolled-up data i.e. not representing the actual count), this is a configurable parameter and can be set in KPI definition while adding/editing a KPI.
```

Comment:
```suggestion
- Druid based KPIs require a name for the <strong>Count Column</strong> in case of rolled-up data (this column can also be proxy for count for the rolled-up data i.e. not representing the actual count), this is a configurable parameter and can be set in KPI definition while adding/editing a KPI.
```